src/current/_includes/releases/withdrawn.md

```diff
@@ -0,0 +1,5 @@
+{% if release.withdrawn == "true" %}
```

Comment:
I moved this into an include to match how the test release admonition include is handled.